state/src/impls/top_level.rs

```diff
@@ -1158,7 +1230,7 @@ mod tests_parcel {
                 expected: 0,
                 found: 2
             }))),
-            state.apply(&parcel, &sender_public, &get_test_client())
+            state.apply(&parcel, &H256::zero(), &sender_public, &get_test_client())
```

Comment:
How about using `H256::random()` for parcel_hash?

Reply:
Fixed